packages/a/aws-sdk-cpp/xmake.lua

```diff
@@ -3,14 +3,16 @@ package("aws-sdk-cpp")
     set_description("AWS SDK for C++")
 
     add_urls("https://github.com/aws/aws-sdk-cpp.git")
-    add_versions("1.9.333", "6ec095fa11379174da20bf1523c93c92503eb8e2")
+    add_versions("1.9.362", "e9372218a2c8fab756ecaa6e4fefcdb33c3670c1")
 
     add_configs("build_only", {description = 'By default, all SDKS are built, if only AWS S3 is required, then set build_only="s3", with multiple SDKS separated by commas.'})
     add_deps("libcurl", "openssl", "zlib")
     add_deps("cmake")
 
     on_install("linux", "macosx", function (package)
         local configs = {}
+        table.insert(configs, "-DMINIMIZE_SIZE=ON")
+        table.insert(configs, "-DCMAKE_PREFIX_PATH=" .. package:installdir())
```

Comment:
why? xmake will pass all deps path to CMAKE_PREFIX_PATH in cmake.install

Reply:
openssl already included, but has "missing: crypto_LIBRARY" error.
Okay, Let me see again:)

Reply:
so you can patch it to disable find crypto and set packagedeps, it will force pass all flags about openssl

Reply:
> so you can patch it to disable find crypto and set packagedeps, it will force to pass all flags about openssl

Ok, I will patch Findcrypto.cmake file